Sources/TestSupport/SwiftPMProduct.swift

```diff
@@ -84,4 +89,16 @@ public enum SwiftPMProduct {
             throw error
         }
     }
+
+    public static func packagePath(for packageName: String, packageRoot: AbsolutePath) throws -> AbsolutePath {
+        // FIXME: This uses package tool right now, and should be replaced by --get-package-path
+        // whenever we design that.
+        try POSIX.chdir(packageRoot.asString)
+        let packageTool = SwiftPackageTool()
```

Comment:
Actually it be better to just assume that we know the paths because we also need to test things like clean, which will need knowledge of where the build folder is (not just checkout folder)